content/api/commands/clock.md

```diff
@@ -236,15 +236,20 @@ not override the time functions of any `iframe` embedded on the page.
 
 ### Behavior
 
-#### clock behavior before `cy.visit()`
+#### clock behavior before<br />`cy.visit()` or `mount(<MyComponent />)`
```

Comment:
Instead of adding `mount()` here, why not revert this back to what it was and add a section immediately after called

### clock behavior before `mount()`

Reply:
👍